### DISCUSSION TITLE
Ignore indexer Property with automapper 11

### DISCUSSION THREAD

**Discussion by jogibear9988** [LuckyPennySoftware/AutoMapper#3858] [NA]
In your upgrade Guide here:
https://github.com/AutoMapper/AutoMapper/blob/master/docs/11.0-Upgrade-Guide.md#c-indexers-item-property

You wrot eIndexers are not Ignored by default.

But how can I ignore them?
I tried per Type via:

     cfg.CreateMap<T,T>().ForMember(...);

But the "Item" Property is not visible in C#

How should it be done per member?

**Comment by lbargaoanu** [comment]
https://github.com/AutoMapper/AutoMapper/issues/3597
